tests/backends/test_redis.py

```diff
@@ -47,3 +60,9 @@ def test_session_is_modified(redis_session: Session) -> None:
 
     redis_session["key"] = "value"
     assert redis_session.is_modified is True
+
+
+def test_improperly_configured_redis_key() -> None:
+    url = os.environ.get("REDIS_URL", "redis://localhost")
+    with pytest.raises(AssertionError):
+        RedisBackend(url, redis_key_func="a_random_string")  # type: ignore[arg-type]
```

Comment:
specifically ignoring that mypy error since we indeed pass it wrongly to test the assertion error